Description: Fix uninitialised value for render_cond.
 gsicc_get_srcprofile doesn't return a render_cond value in all cases.
 This leads to an undefined value being read,
 and this can lead to a SEGV.
Origin: upstream, http://git.ghostscript.com/?p=ghostpdl.git;h=efe6d83
Author: Robin Watts <Robin.Watts@artifex.com>
Forwarded: yes
Bug: http://bugs.ghostscript.com/show_bug.cgi?id=699809
Last-Update: 2018-10-18
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/base/gsicc_cache.c
+++ b/base/gsicc_cache.c
@@ -644,6 +644,7 @@
 {
     (*profile) = NULL;
     (*render_cond).rendering_intent = gsPERCEPTUAL;
+    (*render_cond).cmm = gsCMM_DEFAULT;
     switch (graphics_type_tag & ~GS_DEVICE_ENCODES_TAGS) {
     case GS_UNKNOWN_TAG:
     case GS_UNTOUCHED_TAG:
